relayer/cmd/generate_beacon_data.go

```diff
@@ -38,7 +38,9 @@ func generateBeaconDataCmd() *cobra.Command {
 	}
 
 	cmd.Flags().String("url", "http://127.0.0.1:9596", "Beacon URL")
-	cmd.Flags().Bool("wait_until_next_period", true, "Waiting until next period")
+	cmd.Flags().Bool("wait_until_next_period", false, "Waiting until next period")
```

Comment:
I think we should leave the default true here. The reason being is if you update all the other fixtures, but leave our regenerating the next sync committee files, the tests will likely fail because the tests will be for different runs.

The beacon test fixtures and the inbound message fixtures should be in the same run, since I need to the beacon imports + message test in the same test.

Reply:
Reverted in https://github.com/Snowfork/snowbridge/pull/1125/commits/98b37582c0a63a324c203556581238e7b6e1cbf6

Btw: WDYT if we totally removed the fixtures/tests for next period, it's a long wait time here.  Since test fixtures in current period are also able to cover the switch-over process so coverage will not be decreased a lot? 

Anyway, not the scope of this PR and we can improve it later.